src/CBot/CBotEnums.h

```diff
@@ -245,16 +248,15 @@ enum CBotError : int
     CBotErrNoRetVal      = 6003, //!< function did not return results
     CBotErrNoRun         = 6004, //!< Run() without active function
     CBotErrUndefFunc     = 6005, //!< calling a function that no longer exists
-    CBotErrNotClass      = 6006, //!< this class does not exist
```

Comment:
I would prefer avoiding changing the associated numbers for these errors if possible. Usually it's just because it generates a lot of mess in the commit diff, but in this case it's also because of an undocumented feature which allows you to catch these errors using undocumented try...catch construct by their IDs. I guess we should finally decide what to do with try..catch in cbot at some point...